admin/templates/base.html

```diff
@@ -26,6 +26,23 @@
         <script src="/html5-shiv/html5shiv.js"></script>
         <![endif]-->
         <meta property="og:image" content="/assets/images/govuk-opengraph-image.png">
+        <style>
+            .govuk-header__navigation {
+                display: block !important;
```

Comment:
Is !important needed? Should be avoided if possible

Reply:
In this particular case it is, because we're overriding a style set in the standard Design System CSS file.
But, you're right about trying to avoid it - I've used it a few times in the file, but it's only really needed once - I'll fix that :-)